src/server/cluster/cluster_family.cc

```diff
@@ -691,16 +697,15 @@ void ClusterFamily::MigrationConf(CmdArgList args, ConnectionContext* cntx) {
   VLOG(1) << "Create slot migration config";
   CmdArgParser parser{args};
   auto port = parser.Next<uint16_t>();
-  (void)port;  // we need it for the next step
+  ;
 
   std::vector<ClusterConfig::SlotRange> slots;
   do {
     auto [slot_start, slot_end] = parser.Next<SlotId, SlotId>();
     slots.emplace_back(SlotRange{slot_start, slot_end});
   } while (parser.HasNext());
 
-  if (auto err = parser.Error(); err)
-    return cntx->SendError(err->MakeReply());
+  DCHECK(!parser.Error());
```

Comment:
why did you changed this to DCHECK?

Reply:
it's internal command, so I've decided that there is no need to send error

Reply:
I think we better have an error returned here as if for some reason we have an error here in production we want to get the error on the target node and cancel the migration